cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/FieldLineageSummary.java

```diff
@@ -24,28 +26,29 @@
 
 /**
  * Represents the lineage summary associated with the field of a dataset.
- * Summary in the backward direction includes the information about the fields
```

Comment:
Represents the lineage summary associated with the field of a dataset.

- In incoming direction, it consists of the datasets and their fields ({@link DatasetField}) that this field originates from;
- In outgoing direction, it consists of the datasets and their fields ({@link DatasetField}) that were computed from this field.

Reply:
done.